src/jesse.erl

```diff
@@ -84,25 +83,21 @@ del_schema(Key) ->
 
 %% @doc Loads schema definitions from filesystem to in-memory storage.
 %%
-%% Equivalent to `load_schemas(Path, ParserFun, ValidationFun, MakeKeyFun)'
-%% where `ValidationFun' is `fun jesse_json:is_json_object/1' and
-%% `MakeKeyFun' is `fun jesse_lib:get_schema_id/1'. In this case
-%% the key will be the value of `id' attribute from the given schemas.
+%% Equivalent to `load_schemas(Path, ParserFun, ValidationFun)'
+%% where `ValidationFun' is `fun jesse_json:is_json_object/1'.
 -spec load_schemas( Path      :: string()
                   , ParserFun :: fun((binary()) -> json_term())
                   ) -> jesse_database:update_result().
 load_schemas(Path, ParserFun) ->
   load_schemas( Path
               , ParserFun
               , fun jesse_lib:is_json_object/1
-              , fun jesse_lib:get_schema_id/1
```

Comment:
@lazedo how does your `get_schema_id` look like?

It's been a while since I filed this PR, but the idea is 1) either your schema file has an `id`, 2) either it doesn't and then we default the id to the path of the schema file itself. FYI the schema can be referenced from another schema both by it's id and it's filesystem location.

Let me know what's your usecase, and we can figure out smth.